includes/admin/class-wc-rest-stripe-settings-controller.php

```
@@ -630,18 +642,18 @@ private function update_oc_settings( WP_REST_Request $request ) {
 	 * @param array $payment_method_ids_to_enable The list of payment method ids to enable.
 	 * @param bool  $is_upe_enabled               Whether UPE is enabled.
 	 *
-	 * @return void
+	 * @return true|\WP_Error|null True on success, WP_Error on failure, null if skipped.
 	 */
 	private function update_enabled_payment_methods( $payment_method_ids_to_enable, $is_upe_enabled ) {
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $payment_method_ids_to_enable. Keep variable name length under 20. (undefined)

(LongVariable)